src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area,Drawable {
+    private final String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public String draw() {
+        return ", area: " + getArea() + ", color: " + color;
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
```

Comment:
Returning a default value of 0 for `getArea` is not a good practice, as this method should be abstract and implemented by subclasses to provide their specific area calculations.